package.json

```diff
@@ -37,10 +37,15 @@
   },
   "packageManager": "yarn@3.2.3",
   "devDependencies": {
+    "@types/react": "^18.3.5",
     "husky": "^8.0.1",
     "lint-staged": "^13.0.3"
   },
   "engines": {
     "node": ">=18.18.0"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
+    "react-icons": "^5.3.0"
```

Comment:
These are not needed here, as they belong and are in the nextjs folder package.json.